linux/test/README.md

```diff
@@ -175,21 +175,13 @@ For example:
     OMEROVER=OMERO-DEV-latest ./docker-build.sh ubuntu1404_nginx
 
 The supported values are: 
-OMERO-DEV-latest, OMERO-DEV-merge-build, 5.0 (latest i.e. 5.0.8), 5.1 (latest), 5.2 (latest), latest (default)
-
-If you select 5.0, you will have to apply a Java security patch
-see http://blog.openmicroscopy.org/tech-issues/2015/07/21/java-issue/
-for information and instructions.
-
-5.1 and 5.0 only work with nginx.
+OMERO-DEV-latest, OMERO-DEV-merge-build,s 5.2 (latest), latest (default)
```

Comment:
I guess the 's' just slipped in as typo? But still I don't quite understand, 5.2 is not latest.